### PR TITLE
Fix date filter for YYYY-MM-DD

### PR DESCRIPTION
**Pull request by cat-bro**
When updating a filter from a loaded favourite, the date might be stored in YYYY-MM-DD format

### CODE REVIEW

**Comment by cat-bro** [comment]
We were using the string for the value prop before #33

File: src/filters/types/Date.js [hunk 1]
```diff
@@ -79,11 +79,12 @@ class DateComponent extends React.Component {
 
   render() {
     const { value, inputFormat, relativeDateOptions, comparison, fixedComparison } = this.props
+    const date = value === null ? '' : moment(value).format(inputFormat)
     let dateChoice
     if (comparison === fixedComparison.value) {
       dateChoice = (
         <DayPickerInput
-          placeholder={value === null ? '' : moment(value).format(inputFormat)}
+          placeholder={date}
```
Comment (scaredcat) [inline src/filters/types/Date.js]: we can leave this as
```javascript
value ? date : ''
```
if we don't want placeholder text to appear when the value is empty

Comment (cat-bro) [reply src/filters/types/Date.js]: yes that's fair enough

Comment (cat-bro) [reply src/filters/types/Date.js]: no that's exactly the same I think

**Pull request merged**